sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -34,6 +35,50 @@
 # This module is experimental. No backwards-compatibility guarantees.
 
 
+class StatusServer(object):
+
+  @classmethod
+  def get_thread_dump(cls):
+    lines = []
+    frames = sys._current_frames()  # pylint: disable=protected-access
+
+    for t in threading.enumerate():
+      lines.append('--- Thread #%s name: %s ---\n' % (t.ident, t.name))
+      lines.append(''.join(traceback.format_stack(frames[t.ident])))
+
+    return lines
+
+  def start(self, STATUS_HTTP_PORT=0):
```

Comment:
if `STATUS_HTTP_PORT` is not going to be a constant, we should use the `status_http_port` convention we normally use for args.

Reply:
Makes sense.